cmd/translatesfx/translatesfx/otel_test.go

```diff
@@ -399,219 +394,6 @@ func TestSAExcludesToExpr_MetricNamesAndDimensions(t *testing.T) {
 	assert.Equal(t, expected, ex[0])
 }
 
-func TestFilterTranslation(t *testing.T) {
```

Comment:
I think these tests confirm that sa `metricsToInclude` and `metricsToInclude` filters are correctly translated via `saExcludesToExpr` to the transform process expr rules. Why are they being removed?

Reply:
Sorry, I'll take another look.

Reply:
OK, that's because they are used by translatesfx. I missed that. I will close this PR.

Reply:
I think the required agent method was inlined in translatesfx to avoid exporting it and you may be able to just drop any changes to that cmd in this PR. whatever works though